row_cache.cc

```diff
@@ -43,9 +43,17 @@ logging::logger clogger("cache");
 using namespace std::chrono_literals;
 using namespace cache;
 
+static schema_ptr to_read_domain(const query::partition_slice& slice, schema_ptr table_domain_schema) {
```

Comment:
Is `read domain` the same thing as `query domain`? If so, why use different names?

Reply:
It is the same, I'll switch to "query domain".